### DISCUSSION TITLE
Open Source License Compliance Issues

### DISCUSSION THREAD

**Discussion by mrhello369** [cloudscape-design/components#1681] [Q&A] [answered]
First of all, thanks to all the contributors for their efforts, this is a very helpful project to help us build various UI interfaces quickly.

I use this project in my project (KubeFin: https://github.com/kubefin/kubefin): https://github.com/cloudscape-design/demos/

Do I use it this way to comply with the open source LICENSE specification:
1. Each native file maintains the Copyright statement.
2. Refer to the address of the project used in the project. https://github.com/kubefin/kubefin/tree/main/dashboard
3. Keep the LICENSE file in the corresponding folder. https://github.com/kubefin/kubefin/blob/main/dashboard/web/LICENSE

Thanks to the contributors!

**Comment by mrhello369** [comment]
@johannes-weber Can u help me about this? Thanks a lot.

**Comment by jperals** [comment, marked as answer]
Hello,

Sorry for the delay in the answer, and glad that you find Cloudscape useful!

Essentially none of the three points that you describe above is really mandatory for you. Our demos code is licensed under the [MIT-0](https://github.com/aws/mit-0) license so that code can be freely copied and used with the least friction possible, without attribution or any other obligation. But thanks for asking!

**Reply by mrhello369** [reply]
Hi, @jperals , Thanks for classification!
